app/assets/stylesheets/application.css.scss

```diff
@@ -15,8 +15,29 @@
  *= require foundation_and_overrides
  */
 
+.logo {
+  background-color: #d9b82d;
+  border-color: #111111;
```

Comment:
Color literals like `#111111` should only be used in variable declarations; they should be referred to via variable everywhere else.